ee/clickhouse/views/insights.py

```diff
@@ -107,8 +107,7 @@ def calculate_funnel(self, request: Request) -> Dict[str, Any]:
                 ).run()
             }
         else:
-            base_uri = request.build_absolute_uri("/")
-            return {"result": funnel_order_class(team=team, filter=filter, base_uri=base_uri).run()}
+            return {"result": funnel_order_class(team=team, filter=filter).run()}
```

Comment:
I guess this is just to be able to debug the issue? Would you suggest reverting this change before merge or leaving in?

Reply:
I was thinking that since it works without the base_uri being set. It's nicer to have all of the ways that we create `funnel_order_class` be more similar

Reply:
(open to being challenged on that!)